packages/bpk-component-card/src/BpkDividedCard.figma.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import figma from '@figma/code-connect';
+
+import BpkDividedCard, { ORIENTATION } from './BpkDividedCard';
+
+figma.connect(
+  BpkDividedCard,
+  'https://www.figma.com/design/irZ3YBx8vOm16ICkAr7mB3/Backpack-Components?node-id=22678%3A26157',
+  {
+    props: {
+      orientation: figma.enum('Size', {
+        Desktop: ORIENTATION.horizontal,
+        Mobile: ORIENTATION.vertical,
+      }),
+    },
+    example: ({ orientation }) => (
+      <BpkDividedCard
+        orientation={orientation}
+        primaryContent="Primary content"
+        secondaryContent="Secondary content"
```

Comment:
Requires Design work to allow primaryContent secondaryContent orientation isElevated to be set. Record in doc.